velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -904,6 +904,19 @@ struct DateTruncFunction : public TimestampWithTimezoneSupport<T> {
     }
   }
 
+  /// For fixed interval like second, minute, hour, day and week
+  /// we can truncate date by a simple arithmetic expression:
+  /// floor(seconds / intervalSeconds) * intervalSeconds.
+  FOLLY_ALWAYS_INLINE Timestamp
+  adjustByArithmetic(long seconds, long intervalSeconds) {
```

Comment:
1. Move to `private:`
2. Rename to `adjustEpoch`
3. Use `int64_t` instead of `long`